eclass/cargo.eclass

```diff
@@ -146,7 +146,7 @@ cargo_src_compile() {
 
 	export CARGO_HOME="${ECARGO_HOME}"
 
-	cargo build -j $(makeopts_jobs) $(usex debug "" --release) "$@" \
+	cargo -vv build -j $(makeopts_jobs) $(usex debug "" --release) "$@" \
```

Comment:
can you imagine to make the -vv optional?

Reply:
yes, changes coming soon.
default will be verbose though.
but it'll be possible to override it back to quiet.
also NOCOLOR will be supported.

Reply:
https://github.com/gentoo/gentoo/pull/14135